### PR TITLE
add each subsequent subdivision to the end

### PR DESCRIPTION
**Pull request by regineheberlein**

### CODE REVIEW

**Review by christinach — COMMENTED** [review]
@regineheberlein there is a rubocop error. you can fix it by adding in https://github.com/pulibrary/aspace_helpers/blob/fix_subdivisions/.rubocop.yml 

Metrics/AbcSize:
  Exclude:
    - 'reports/aspace2alma/get_MARCxml.rb'

**Comment by regineheberlein** [comment]
Thanks @christinach , doing it now!

File: .rubocop.yml [hunk 1]
```diff
@@ -3,3 +3,9 @@ inherit_from: .rubocop_todo.yml
 AllCops:
   TargetRubyVersion: 3.1
   NewCops: enable
+  
+Metrics/AbcSize:
+Exclude:
+- 'reports/aspace2alma/get_MARCxml.rb'
```
Comment (christinach) [inline .rubocop.yml:9]: line 9 needs indentation. one space to the right.

Comment (christinach) [reply .rubocop.yml]: @regineheberlein sorry two spaces :)

**Review by christinach — APPROVED** [review]

**Pull request merged**